pom.xml

```diff
@@ -27,7 +27,7 @@
 
         <!-- Library Versions -->
         <spring.version>4.3.18.RELEASE</spring.version>
```

Comment:
Can we make this 4.3.24 so that spring-security and spring versions are the latest before they moved to version 5?